sites/cheerpj/src/content/docs/13-tutorials/05-interoperability-tutorial.mdx

```diff
@@ -0,0 +1,159 @@
+---
+title: Java and JavaScript Interoperability
+description: Learn how to enable Java and JavaScript communication using CheerpJ.
+---
+
+With CheerpJ, a Java application can integrate with JavaScript in a browser environment, enabling seamless interaction between Java and JavaScript code.
+
+In this tutorial, we’ll explore creating a two-way communication channel between Java and JavaScript using CheerpJ. The idea is to allow a Java class to access JavaScript functions and vice versa, to enable sending data between the two environments.
+
+## Prerequisites
+
+{/* - [Download the template project](/docs/cheerpj3/tutorials/CheerpJInteroperabilityTutorial.zip) and unzip it. */}
+
+- [Download the application files](/docs/cheerpj3/tutorials/CheerpJInteroperabilityTutorial.zip) and extract them.
+- [Node.js](https://nodejs.org/en/) (>= 18)
+
+## Project Structure
+
+The project consists of the following files and directories:
+
+```plaintext
+CheerpJInteroperbailityTutorial/
+├── com/example
+│   └── Example.java
+│   └── manifest.txt
+├── index.html
+└── example.jar
+└── Makefile
+```
```

Comment:
The folder also contains the `Example.class` files, we should add that to the tree here too.